src/amo/actions/featured.js

```diff
@@ -0,0 +1,27 @@
+import {
+  FEATURED_GET,
+  FEATURED_LOADED,
+  FEATURED_FAILED,
+} from 'core/constants';
+
+
+export function featuredGet({ addonType }) {
```

Comment:
I find it makes the most sense to read action functions as verbs. Perhaps `getFeatured` ?